R/report_descriptives.R

```diff
@@ -0,0 +1,18 @@
+# A funtion to take a seminr model and return item and construct descriptives
+get_descriptives <- function(seminr_model, na.rm = TRUE) {
```

Comment:
no need for prefix `get_` in function names. i think it's obvious that `descriptives()` would return descriptives.

Reply:
Done.